2019_03_22_InputOutputHome/src/Main.java

```diff
@@ -0,0 +1,147 @@
+import org.apache.commons.io.FileUtils;
+
+import java.io.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Main {
+	public static void main(String[] args) throws IOException {
+		List<Customer> customers = Arrays.asList(new Customer("Ivan","Ivanich", 22),
+				new Customer("Sergio", "Pedro", 25),
+				new Customer("Ishtvan", "Poladi", 79));
+
+
+
+		System.out.println("******************Just reading file***************************************");
+		FileReaderFromFile("/home/sabadosh/Documents/test.txt");
+
+		System.out.println("******************Summing all integers in file****************************");
+		sumIntFromFile("testWithIntegers");
+
+		System.out.println("******************Sorting array with strings *****************************");
+		System.out.println("BEFORE:");
+		FileReaderFromFile("StringsWithDifferentLength");
+		System.out.println("");
+		System.out.println("After: ");
+		sortingStrings("StringsWithDifferentLength");
+
+		System.out.println("******************Sorting CSV file****************************************");
+		sortingCSVfile("CSVfile");
+
+		System.out.println("*******************Save list in CSV File**********************************");
+		makeCSVFile(customers, "Customers");
+
+		System.out.println("********************Copy file*********************************************");
+		copyFile("Customers", "/home/sabadosh/Documents/copyCustomers.txt");
+
+		System.out.println("********************Summing gadgets price*********************************");
+		summingGadgetsPrice("NamesWithGadgets");
+
+
+
+
+	}
+
+	public static void FileReaderFromFile(String fileAddress) {
+		try (FileReader reader = new FileReader(fileAddress)) {
+			int c;
+			while ((c = reader.read()) != -1) {
+				System.out.print((char) c);
+			}
+		} catch (IOException ex) {
+			ex.printStackTrace();
+		}
+	}
+
+	public static void sumIntFromFile(String fileAddress) {
+		int res;
+		try (BufferedReader br = new BufferedReader(new FileReader(fileAddress))) {
+			res = br.lines().mapToInt(Integer::parseInt).sum();
+			System.out.println("Sum is " + res );
+		} catch (Exception ex) {
+			ex.printStackTrace();
```

Comment:
What is the use of this stack trace? If I call this function, how do I know that it works?